plugins/linuxv2/model/namespace/namespace.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package namespace;
+
+message Namespace {              /* Linux network namespace */
+    enum NamespaceType {
+        PID_REF_NS = 0;          /* namespace of a given process */
+        MICROSERVICE_REF_NS = 1; /* namespace of a docker container running given microservice */
+        NAMED_NS = 2;            /* named namespace */
+        FILE_REF_NS = 3;         /* namespace referenced by a file handle*/
+    }
```

Comment:
Enum definitions which are defined inside message should be defined just before first use as field in that message. So here, it should be just before the `type`.